frontend/src/components/teacher/session-creation/DistributeAssessmentButton.tsx

```diff
@@ -29,6 +34,11 @@ const DistributeAssessmentButton = ({
   }>(CREATE_TEST_SESSION, {
     refetchQueries: [{ query: GET_TEST_SESSIONS_BY_TEACHER_ID }],
   });
+  const [updateSession] = useMutation<{
+    updateSession: string;
+  }>(UPDATE_TEST_SESSION, {
+    refetchQueries: [{ query: GET_TEST_SESSIONS_BY_TEACHER_ID }],
```

Comment:
nit: we should change this and the one above to use `getQueryName` so we can avoid worrying about extra parameters in the future